src/components/Badge/index.tsx

```diff
@@ -0,0 +1 @@
+export { Badge } from "./Badge";
```

Comment:
I have already did this file inside the PR https://github.com/casavo/habitat/pull/25